scripts/model_gen_ng/model_gen_ng/templates/class_forward_decls.tpl

```diff
@@ -0,0 +1,8 @@
+// Forward declarations
+[% for ns, names in ns_to_names.items() %]
+namespace [[ ns ]] {
+[%- for name in names|sort %]
+class [[ name ]];
+[%- endfor %]
+} // namespace [[ ns ]]
+[% endfor %]
```

Comment:
Testing highlighting:
```c++
/* for ns, names in ns_to_names.items() */
namespace /* ns */ {
/*- for name in names|sort */
class /* name */;
/*- endfor */
} // namespace /* ns */
/* endfor */
```
This is kinda nice.

Reply:
I'm evil-laughing for this terrible but good idea I had. I changed the tokens to
```cpp
/*# This is a comment #*/
// Forward declarations
/** for ns, names in ns_to_names.items() **/
namespace /*= ns =*/ {
/**- for name in names|sort **/
class /*= name =*/;
/**- endfor **/
} // namespace /*= ns =*/
/** endfor **/
```
The tokens for delimiting a "block" and a variable to print have to be different. Also, while just `/*` for output (and `/**` for blocks) works, I find it confusing, so I used `/*=`